app/middleware/locationValidator.js

```diff
@@ -30,4 +31,20 @@ function validateLocation(req, res, next) {
   }
 }
 
+function renderNoResultsPage(req, res) {
+  /* eslint-disable no-param-reassign */
+  res.locals.nearbyServices = [];
+  res.locals.openServices = [];
+  /* eslint-enable no-param-reassign*/
+  renderer.results(req, res);
+}
+
+function validateLocation(req, res, next) {
+  if (isNotEnglishLocation(res.locals.location)) {
+    renderNoResultsPage(req, res);
```

Comment:
Can a log statement of what the postcode was be added please? It would be good to have a record of what has been added.